deploy/demo.jsonnet

```diff
@@ -0,0 +1,53 @@
+(import 'cortex.jsonnet') +
+(import 'lib/dynamodb.libsonnet') +
+(import 'lib/prometheus.libsonnet') +
+(import 'lib/nginx.libsonnet') + 
+{
+    _config+:: {
+        local dynamodb_uri = 'dynamodb://user:pass@' + $._config.dynamodb.name + '.' + $._config.namespace + '.svc.cluster.local:8000',
```

Comment:
this way of passing credentials fails for me.  I have to set them via `envKVMixin`

Reply:
I'm interested to hear why this didn't work. I'm able to build the demo without any problems with it setup this way. What are you seeing when you try to build this?